rock.osdeps

```diff
@@ -411,3 +411,12 @@ gawk:
 
 bzip2:
     debian,ubuntu: libbz2-dev
+
+slam/pcl:
+    ubuntu:
+        default: libpcl-dev
+        '14.04': libpcl-1.7-all-dev
```

Comment:
libpcl-1.7-all-dev does not exist on Ubuntu 14.04, so it does require to add a PPA?!